specification/mysql/resource-manager/readme.python.md

```diff
@@ -23,10 +23,20 @@ Generate all API versions currently shipped for this package
 ```yaml $(python)
 clear-output-folder: true
 batch:
-  - tag: package-flexibleserver-2023-12-01-preview
+  - tag: package-flexibleserver-2023-12-30
   - tag: package-2020-01-01
 ```
 
+### Tag: package-flexibleserver-2023-12-30 and python
+
+These settings apply only when `--tag=package-flexibleserver-2023-12-30 --python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+``` yaml $(tag) == 'package-flexibleserver-2023-12-30' && $(python)
+namespace: azure.mgmt.rdbms.mysql_flexibleservers
+output-folder: $(python-sdks-folder)/rdbms/azure-mgmt-rdbms/azure/mgmt/rdbms/mysql_flexibleservers
+```
+
 ### Tag: package-flexibleserver-2023-12-01-preview and python
```

Comment:
@ChenxiJiang333  remove line40~58 which is useless configuration.

Reply:
removed